extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -117,7 +117,7 @@ function () {
 				$dateTwoYearsAgo = date( 'Y-m-d', strtotime( '-2 years' ) );
 
 				$sqlData = ( new WikiaSQL() )
-					->SELECT( 'rev_user_text, rev_user, wup_value' )
+					->SELECT( 'rev_user_text, rev_user, wup_value, MAX(rev_timestamp) AS latest_revision' )
```

Comment:
Do you still need `wup_value` if you replaced # of contributions with last revision date?